mobile/src/home-view.js

```diff
@@ -114,6 +116,20 @@ class HomeView extends Component {
           }
         }
       })
+      anomRef.on('child_added', data => {
+        this.setState({ anom: [...this.state.anom, {...data.val(), key: data.key }] })
+      })
+
+      anomRef.on('child_changed', data => {
+        var anom = this.state.anom
```

Comment:
No reason to use `var`.  `const` in this case, or `let` when the value may change.

Reply:
Since its an array we can use const because only the objects variables are changing?